modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -24,82 +24,80 @@ This page describes the options on the *Deploy Application* panel that is identi
 
-If you created an application in a sandbox environment and tested it, you would eventually want to migrate it to production.
-
-To do so:
+To move an app from a sandbox environment to production:
 
-. Logged into your Anypoint Platform account, go to your production environment
-. In the *Applications* tab, click the *Deploy Application* button
+. Log into your Anypoint Platform account, and go to your production environment.
+. In the *Applications* tab, click the *Deploy Application* button.
 +
-image::deploying-to-your-own-servers-d84f7.png[]
+// image::deploying-to-your-own-servers-d84f7.png[]
 
-. Then when picking the application file, instead of uploading a new file, click *Get from sandbox*.
+. When choosing the application file, click *Get from sandbox*.
 +
 image::deploying-to-cloudhub-d6f7a.png[]
 
-. You can then choose between your sandbox environments, and select an application to deploy in production. Select an application and click *Apply*
+. In the *Get from sandbox* dialog window, select your sandbox environment, then select an application to deploy in production, and click *Apply*.
 
-. Continue setting up your deployment as you would with any normal deployment
+. Continue setting up your deployment as you would with any other deployment.
 
 
 ==== Limitations
 
 * Only a *CloudHub* Administrator can move applications between environments.
-* To avoid domain name conflict, an application cannot exist in two environments at the same time. Alter the application's name slightly to deploy it to another environment.
-* You cannot move an application into the same environment it's in already. Runtime Manager prevents you from duplicating applications in a single environment. If you wish to have the same, or similar applications in a single environment, modify one application's file to distinguish it from the original.
-
-
-
-
-
-
+* To avoid domain name conflict, an application with the same name cannot be deployed in two environments at the same time. Modify the application's name slightly to deploy it to another environment.
+* Runtime Manager prevents you from duplicating applications in a single environment, so you cannot move an application into the environment it's already in. To have the same, or similar, applications in a single environment, modify one application's file to distinguish it from the original.
 
 
 === From Anypoint Studio
 
-You can easily deploy your applications to CloudHub, straight from Anypoint Studio. This is specially helpful if you're still developing the application and want to deploy it often to an online test environment.
+You can deploy your applications to CloudHub from Anypoint Studio. This is helpful if you're still developing the application and want to deploy it often to an online test environment.
+
+. In Anypoint Studio open your application as a Mule Project.
```

Comment:
Should there be a comma after In Anypoint Studio to prevent "garden path" problem (reading this as if Anypoint Studio modifies "open" instead of being a separate clause)? Just me?